int64 Pointer
int64 NumberOfLines
float32 velocity
time timestamp
